doc/command-line.md

```diff
@@ -993,6 +993,39 @@ size: 5177
 signature license: CC0
 ```
 
+### `sourmash signature fileinfo` - display a summary of the contents of a sourmash collection
+
+Display signature file, database, or collection.
+
+For example,
+```
+sourmash sig fileinfo tests/test-data/prot/all.zip
+```
+will display:
+```
+path filetype: ZipFileLinearIndex
+location: /Users/t/dev/sourmash/tests/test-data/prot/all.zip
+is database? yes
+has manifest? yes
+is nonempty? yes
+num signatures: 8
+** examining manifest...
+31758 total hashes
```

Comment:
all other info is `label: #`, rather than `# label`.

Doesn't really matter - looks fine this way too 🤷🏻‍♀️

Reply:
> Closes #1190, #1439

hah, hadn't even noticed 🤣

Reply:
fixed in 56a678e7, thx! consistency is good, tho people should use JSON instead of parsing this output 👀